ComparisonWithOrleans.md

```diff
@@ -0,0 +1,203 @@
+# Orleans and Akka Actors: A Comparison
+
+## Step-By-Step Review
+
+This overview walks along the Technical Report [MSR-TR-2014-41](http://research.microsoft.com/apps/pubs/default.aspx?id=210931) by *Philip A. Bernstein, Sergey Bykov, Allan Geller, Cabriel Kliot, Jorgen Thelin (Microsoft Research)*, for another comparison focusing on Erlang and Riak see [Christopher Meiklejohn’s excellent article](http://christophermeiklejohn.com/orleans.html).
+
+### Introduction
+
+Different focus:
+
+  * Akka as a toolkit for building distributed systems, offering the full power but also exposing the inherent essential complexity.
+  
+  * Orleans restricts applicability in order to allow seamless use without understanding distributed computing.
```

Comment:
I think this sentence is very judgmental. Orleans allows non distributed computing experts to write distributed services, but it does not mean that an expert can not find it useful or necessarily limited in applicability or with less performance. An expert can configure/extend it for a wide range of applications. I would have said: "Orleans simplifies distributed computing allowing non experts to write distributed services. At the same time there is enough extension points and flexibility in Orleans to allow even an expert to customize his services in a flexible way.